react-native-purchases-ui/android/src/main/java/com/revenuecat/purchases/react/ui/PaywallViewManager.kt

```diff
@@ -18,9 +18,12 @@ internal class PaywallViewManager : BasePaywallViewManager<PaywallView>() {
     }
 
     override fun createViewInstance(themedReactContext: ThemedReactContext): PaywallView {
-        return PaywallView(themedReactContext).also { view ->
-            view.setPaywallListener(createPaywallListenerWrapper(themedReactContext, view))
-            view.setDismissHandler(getDismissHandler(themedReactContext, view))
+    return PaywallView(themedReactContext).apply {
+        // Ensure the view is properly initialized before setting listeners
+        post {
```

Comment:
I think this makes sense @RevenueCat/coresdk ?

Reply:
Right, I guess doing it like this we could potentially lose some events if they happen before the view displays. So for example, if for some reason we want to send a dismiss event before the view is drawn... This seems unlikely so it should be ok yeah, we just need to be aware of this behavior.